From: Vanadiae <vanadiae35@gmail.com>
Date: Fri, 29 May 2020 00:42:44 +0200
Subject: background: use the same message string (#307)

Currently, they are two translatable strings that are almost the same, but the quotation mark.

So use the same string for both, with the quotation mark so that the name of the application is well separated from the surrounding message.

For reference, the original string is at line 215.

Origin: upstream, 1.7.2, commit:8a363223e445178e2c5f0a7c65800b075e54ff60
---
 src/background.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/background.c b/src/background.c
index f8bdff1..7e46283 100644
--- a/src/background.c
+++ b/src/background.c
@@ -305,7 +305,7 @@ handle_notify_background (XdpImplBackground *object,
   g_variant_builder_init (&builder, G_VARIANT_TYPE_VARDICT);
   g_variant_builder_add (&builder, "{sv}", "title", g_variant_new_string (_("Background activity")));
 
-  body = g_strdup_printf (_("%s is running in the background."), arg_name);
+  body = g_strdup_printf (_("“%s” is running in the background"), arg_name);
   g_variant_builder_add (&builder, "{sv}", "body", g_variant_new_string (body));
   g_variant_builder_add (&builder, "{sv}", "default-action", g_variant_new_string ("show"));
 
